src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();

        for (int i = 0; i < USE_CONSTANT; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

Create constant for 3 magic number and use it